### ISSUE TITLE
BED - Left Nav Ordering

### ISSUE THREAD

**Issue by janetgatsby** [patpluspun/fmp_au#88]
Patrick,
Somehow the left nav of books has got reordered. I tried fixing on test. but my changes did not update.

Using http://test.fivemile.com.au/admin/structure/menu/manage/menu-catalog
Need the order to be:

New Releases
Brands
Children
Young Adult
Adult
Ebooks

I can change Prod if you want to remind me how to do this. I ordered them 1-6 but it didn't work on Test.

**Comment by patpluspun** [comment]
This was part of the url aliases change to prevent books from having new-release in the url and thus causing infinite loops when New Release was removed from the product.  It should be fixable just by changing the order of terms in the menu.

I went ahead and changed the order in the menu, it should be good now.  For future reference, the page can be accessed here: http://www.fivemile.com.au/admin/structure/menu/manage/menu-catalog
